### DISCUSSION TITLE
API plugins store

### DISCUSSION THREAD

**Discussion by ab-smith** [intuitem/ciso-assistant-community#1750] [Ideas]
Provide a pattern based on the API for the community to build plugins and integrate with the IT ecosystem:
- pull evidences
- others?

Then we review the submissions before they become part of the store

**Comment by nka11** [comment]
Some use-cases for a minimal set of free plugins to provide :
Dispatch tasks (github, jira, gitlab) from missing evidence
Webhooks for : 
   SAST, Pentest tools, SIEM => Security events
   JIRA automation, github, gitlab-ci => push evidence from previously dispatched tasks (bundled with some integration guide)
